lib/pages/home/widgets/DashboardHeader.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/material.dart';
+import '../../../constants/text_constants.dart';
+import '../../../features/auth/auth_repository/schemas/user.dart';
+import '../../../theme/text_styles.dart';
+import '../../EnterExitRoute.dart';
+import '../../profile/profile_page.dart';
```

Comment:
i think better to use absolute imports here for explicitness, especially for the case of going 3 levels up